Pavlova Natalia/src/components/ModalExample/ModalExample.jsx

```diff
@@ -0,0 +1,47 @@
+/* eslint react/no-multi-comp: 0, react/prop-types: 0 */
+import 'bootstrap/dist/css/bootstrap.min.css';
+import React from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
```

Comment:
```suggestion

import Button from 'react-bootstrap/lib/Button';
import Modal from 'react-bootstrap/lib/Modal';
import ModalHeader from 'react-bootstrap/lib/ModalHeader';
import ModalBody from 'react-bootstrap/lib/ModalBody';
import ModalFooter from 'react-bootstrap/lib/ModalFooter';
```
Это поможет webpack в процессе сборки включить только используемую в проекте часть react-bootstrap, а не всю библиотеку целиком